beacon-chain/core/blocks/block_operations.go

```diff
@@ -329,16 +323,26 @@ func casperSlashingPenalizedIndices(slashing *pb.CasperSlashing) ([]uint32, erro
 	return indicesIntersection, nil
 }
 
-func verifySlashableVoteData(votes *pb.SlashableVoteData) error {
-	totalCustody := len(votes.CustodyBit_0Indices) +
-		len(votes.CustodyBit_1Indices)
-	if uint64(totalCustody) > params.BeaconConfig().MaxCasperVotes {
-		return fmt.Errorf(
-			"exceeded allowed casper votes (%d), received %d",
-			params.BeaconConfig().MaxCasperVotes,
-			totalCustody,
-		)
+func verifySlashableVote(votes *pb.SlashableVote) error {
+	emptyCustody := make([]byte, len(votes.CustodyBitfield))
+	if bytes.Equal(votes.CustodyBitfield, emptyCustody) {
+		return errors.New("custody bit field can't all be 0s")
+	}
+	if len(votes.ValidatorIndices) == 0 {
+		return errors.New("empty validator indices")
+	}
+	for i := 0; i < len(votes.ValidatorIndices)-1; i++ {
+		if votes.ValidatorIndices[i] >= votes.ValidatorIndices[i+1] {
+			return errors.New("validator indices not in descending order")
+		}
+	}
+	if len(votes.CustodyBitfield) != mathutil.CeilDiv8(len(votes.ValidatorIndices)) {
+		return errors.New("custody bit field don't match validator indices length")
```

Comment:
use formatting for the errors over here and below so that the error string is more descriptive